tf2onnx/schema.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.schema
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict, OrderedDict
+from onnx import defs
+
+def version_mapping(schemas):
+    """Obtain version to schema mapping."""
+    schemas_with_version = defaultdict(dict)
+    for s in schemas:
+        schemas_with_version[s.name][int(s.since_version)] = s
+    for name, s in schemas_with_version.items():
+        schemas_with_version[name] = OrderedDict(
+            sorted(schemas_with_version[name].items(), key=lambda x: x[0])
+        )
+    return schemas_with_version
+
+
+class ONNXSchema:
+    """Wrapper for ONNX schema"""
+
+    all_schemas = defs.get_all_schemas_with_history()
+    schemas_with_version = version_mapping(all_schemas)
+
+    @staticmethod
+    def get_schema(name, version):
```

Comment:
need to take domain into consideration.
we only deal with onnx for now, may have onnx.ml in the future.
see this for reference https://github.com/onnx/onnx/blob/master/onnx/defs/schema.h#L602-L610